docs/sources/installation/configuration.md

```diff
@@ -471,6 +466,15 @@ Administrators can increase it if they experience OAuth login state mismatch err
 - [LDAP Authentication]({{< relref "../auth/ldap.md" >}}) (auth.ldap)
 - [Auth Proxy]({{< relref "../auth/auth-proxy.md" >}}) (auth.proxy)
 
+### login_cookie_name
+
+The cookie name for storing the auth token, default is `grafana_session`.
```

Comment:
```suggestion
The cookie name for storing the auth token, the default is `grafana_session`.
```

Reply:
There is no a consistent way for the defaults in the page. I've found all the following forms:
- `By default,`
- `Defaults to:`
- `Default is`
- `The default value is`
- `(default true)`
- `Default value is`

let's decide which one we prefer and apply it everywhere.

Reply:
I don't know which one I prefer, I just think "the default is" is correct English and "default is" is not :)

Reply:
I am of the opinion that it should be "the default" rather than "default". Although some of these are purely from an english grammar perspective. Like "by default, it is set to [redacted]" would be a grammatical choice. however-- when given the options of say, 'the default is true' vs 'default is true' or "default value is true" and "the default value is true"-- I will always pick the options with "the".